templates/confidential-computing/index.html

```diff
@@ -217,10 +217,10 @@ <h2>Launch enterprise-grade confidential VMs on all major public clouds</h2>
           <div class="row">
             <div class="col-2 col-medium-3 col-small-2 u-pad-logo">
               <p class="u-no-margin--bottom">
-                {{ image(url="https://assets.ubuntu.com/v1/786c39f9-Google_Cloud_logo.svg",
+                {{ image(url="https://assets.ubuntu.com/v1/b31d530e-google-cloud-high-res.png",
                   alt="Google Cloud logo",
-                  height="22",
-                  width="120",
+                  height="80",
+                  width="600",
```

Comment:
Tried this sizing out locally and it seems to work:
```suggestion
                  height="27",
                  width="239",
```